.pipelines/npm/npm-conformance-tests.yaml

```diff
@@ -347,6 +346,14 @@ jobs:
             then
               runConformanceWindows | tee $npmLogsFolder/conformance-results
               exitCode=$?
+              if [[ $exitCode != 0 ]]; then
+                # capture HNS state
+                curl -LO https://raw.githubusercontent.com/Azure/azure-container-networking/master/debug/windows/npm/win-debug.sh
```

Comment:
todo: try without curling

Reply:
not sure if it's worth it to do trial and error to figure out how to not use curl